nested/azuresearch.json

```diff
@@ -0,0 +1,44 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "moodleCommon": {
+            "metadata": {
+                "description": "Common Moodle values"
+            },
+            "type": "object"
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Search/searchServices",
+            "apiVersion": "2015-08-19",
+            "location": "[resourceGroup().location]",
+            "name": "[parameters('moodleCommon').azureSearchName]",
+            "sku": {
+		"name": "[parameters('moodleCommon').azureSearchSku]"
```

Comment:
Please be mindful with indentation. Many other such misindented blocks around code...